packages/terra-infinite-list/docs/README.md

```diff
@@ -0,0 +1,56 @@
+# Terra Infinite List
+
+The infinite list component provides virtual dom management and data request callbacks to manage large data sets within a list.
+InfiniteList is a controllerd HOC built upon the terra-list component in order to allow for list style and selection patterns.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-infinite-list`
+  - `yarn add terra-infinite-list`
+
+## Usage
+
+```jsx
+import React from 'react';
+import InfiniteList from 'terra-infinite-list';
+
+
+render() {
```

Comment:
Should we perhaps provide a full fleshed component to show developers how it should be used? I feel like if we don't show them the setting of the state they could be confused on how to use the infinite list.